<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LazzyController</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxStatus">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>各模块状态</string>
    </property>
    <widget class="QLabel" name="labelReplayer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>复盘行情</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelWatcher">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>行情接收</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelExecuter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>交易执行</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTrader">
     <property name="geometry">
      <rect>
       <x>166</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>期货量化策略</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelWatcher1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>22</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>不在线</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelReplayer1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>不在线</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelExecuter1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>不在线</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTrader1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>不在线</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelWatcher2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>•</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelExecuter2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>•</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelReplayer2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>•</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTrader2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>•</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelArbitrageur">
     <property name="geometry">
      <rect>
       <x>166</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>期权套利策略</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelArbitrageur_1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>不在线</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelArbitrageur_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>•</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>策略状态监视</string>
    </property>
    <widget class="QComboBox" name="comboBoxId1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>策略名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>交易品种</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInstrument1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>当前仓位</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEnable1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>启用</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDisable1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>停用</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberPosition1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>策略状态监视</string>
    </property>
    <widget class="QComboBox" name="comboBoxId2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>策略名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>交易品种</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInstrument2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>当前仓位</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEnable2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>启用</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDisable2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>停用</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberPosition2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
